dotnet/src/Connectors/Connectors.UnitTests/HuggingFace/TextGeneration/HuggingFaceTextGenerationTests.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.SemanticKernel;
 using Microsoft.SemanticKernel.Connectors.HuggingFace;
 using Microsoft.SemanticKernel.TextGeneration;
+using SemanticKernel.UnitTests;
```

Comment:
Is this change required? Same for other files in `Connectors.UnitTests` project with the same change.

Reply:
I'm sorry it's by accident, I'll correct it.

Reply:
No worries at all :)

Reply:
@dmytrostruk I checked this, and yes, this using is required because I moved HttpMessageHandlerStub class from unit tests project to common directory. dotnet/src/InternalUtilities/test/HttpMessageHandlerStub.cs

Reply:
Makes sense, thanks for clarification!